main.cc

```diff
@@ -554,16 +568,24 @@ usage() noexcept
 {
 	std::cout << "\n"
 		<< "./tls-perf [options] <ip> <port>\n"
-		<< "  -h,--help    Print this help and exit\n"
-		<< "  -d,--debug   Run in debug mode\n"
-		<< "  -l <N>       Limit parallel connections for each thread"
+		<< "  -h,--help	        Print this help and exit\n"
+		<< "  -d,--debug        Run in debug mode\n"
+		<< "  -l <N>            Limit parallel connections for each thread"
 		<< " (default: " << DEFAULT_PEERS << ")\n"
-		<< "  -n <N>       Total number of handshakes to establish\n"
-		<< "  -t <N>       Number of threads"
+		<< "  -n <N>            Total number of handshakes to establish\n"
+		<< "  -t <N>            Number of threads"
 		<< " (default: " << DEFAULT_THREADS << ").\n"
-		<< "  -T,--to      Duration of the test (in seconds)\n"
-		<< "  -c <cipher>  Force cipher choice (default: "
-		<< DEFAULT_CIPHER << ")\n"
+		<< "  -T,--to           Duration of the test (in seconds)\n"
+		<< "  -c <cipher>       Force cipher choice (default "
+		<< "for TLSv1.2: " << DEFAULT_CIPHER_12 << ",\n"
+		<< "                                                 "
+		<< "for TLSv1.3: " << DEFAULT_CIPHER_13 << "),\n"
+		<< "                                                 "
+		<< "or type 'any' to disable ciphersuite restrictions \n"
+		<< "  -tls_13 version>  Set TLS version for handshake: "
```

Comment:
Some misprint, should be `-V`

Reply:
Misprint was fixed. Actually it's long option `--tls <version>` and key `V` is just a key for `getopt_long()` function and nothing more. I didn't want to add a key like `-V` because it's always stands for 'version', 'verbose' even both some times.